components/grade-table.js

```diff
@@ -11,21 +12,65 @@ class GradeTable{
         for(var i=0; i<grades.length; i++){
             var gradesIndex = grades[i];
 
-            var tableRow = document.createElement('tr');
+            this.renderGradeRow(gradesIndex,this.deleteGrade);
 
-            var studentName = document.createElement('td');
-            studentName.textContent = gradesIndex.name;
-            tableRow.appendChild(studentName);
+            if(grades.length === 0){
+                var pEl = document.getElementById('no-grades');
+                pEl.classList.remove("d-none");
+            } 
 
-            var studentCourse = document.createElement('td');
-            studentCourse.textContent = gradesIndex.course;
-            tableRow.appendChild(studentCourse);
+            // tableBodyEl.appendChild(renderRow);
 
-            var studentGrade = document.createElement('td');
-            studentGrade.textContent = gradesIndex.grade;
-            tableRow.appendChild(studentGrade);
+            // var tableRow = document.createElement('tr');
 
-            tableBodyEl.appendChild(tableRow);
+            // var studentName = document.createElement('td');
+            // studentName.textContent = gradesIndex.name;
+            // tableRow.appendChild(studentName);
+
+            // var studentCourse = document.createElement('td');
+            // studentCourse.textContent = gradesIndex.course;
+            // tableRow.appendChild(studentCourse);
+
+            // var studentGrade = document.createElement('td');
+            // studentGrade.textContent = gradesIndex.grade;
+            // tableRow.appendChild(studentGrade);
+
+            // tableBodyEl.appendChild(tableRow);
```

Comment:
Please remove the commented-out code after you're done referencing it. Otherwise, your code will get very time-consuming to read.